clean/mandeville_pd.py

```diff
@@ -16,12 +16,20 @@ def assign_agency(df, year):
 
 def clean_pprr_20():
     return pd.read_csv(data_file_path('mandeville_pd/mandeville_csd_pprr_2020.csv'))\
-        .rename(columns={'annual_salary': 'salary'})\
         .pipe(clean_column_names)\
+        .rename(columns={
+            'recurring_annual_salary': 'salary', 
+            'job_class_desc': 'rank_desc',
+            'badge': 'badge_no',
+            'terminated_date': 'term_date',
+        })\
+        .pipe(clean_dates, ['hire_date', 'term_date'])\
         .pipe(float_to_int_str, ['term_year', 'term_day', 'term_month', 'hire_year', 'hire_day', 'hire_month'])\
         .pipe(standardize_desc_cols, ['rank_desc'])\
         .pipe(set_values, {'salary_freq': salary.YEARLY})\
         .pipe(clean_salaries, ['salary'])\
+        .pipe(clean_races, ['race'])\
+        .pipe(clean_sexes, ['gender'])\
```

Comment:
We don't have "gender" column but we do have "sex" column. Please rename this.

Reply:
Fixed.